src/jit/compiler.h

```diff
@@ -1642,8 +1643,9 @@ struct fgArgTabEntry
     {
 #ifdef FEATURE_HFA
         return HfaTypeFromElemKind(_hfaElemKind);
-#endif // FEATURE_HFA
+#else
         return TYP_UNDEF;
```

Comment:
Did we have an unreachable return and it was not threated as error by CoreCLR compiler?

Reply:
Yes, I hit the same thing in my /W4 PR.